### DISCUSSION TITLE
URL from solo in SkyDewHeaterControl

### DISCUSSION THREAD

**Discussion by ruspa1978** [AllskyTeam/allsky-modules#88] [General]
Hi i have a solo cloudwatcher and i tried to insert in setings url of my solo http://aagsolo, then http://192.168.188.6 and then this http://aagsolo/cgi-bin/cgiLastData but i have a "Failed to read sensor", how can i solve this? Thanks

**Comment by troublestarter** [comment]
Hello
I'm not familiar acutally with this setup, but can you detail a little more your setup ?
Thanks you

**Comment by Alex-developer** [comment]
> Hi i have a solo cloudwatcher and i tried to insert in setings url of my solo http://aagsolo, then http://192.168.188.6 and then this http://aagsolo/cgi-bin/cgiLastData but i have a "Failed to read sensor", how can i solve this? Thanks

I dont have a Solo but if you browse the the Urls provided above what results do you get? The Solo should return json data like

```
{
  "LastReadings": {
    "dataGMTTime": "2023/12/17 22:52:40",
    "cwinfo": "Serial: 2550, FW: 5.89",
    "clouds": -14.85,
    "cloudsSafe": "Unsafe",
    "temp": 8.45,
    "wind": 12,
    "windSafe": "Safe",
    "gust": 13,
    "rain": 3072,
    "rainSafe": "Safe",
    "lightmpsas": 20.31,
    "lightSafe": "Safe",
    "switch": 0,
    "safe": 0,
    "hum": 65,
    "humSafe": "Safe",
    "dewp": 2.25,
    "rawir": -19.15,
    "abspress": 1003.6,
    "relpress": 1032.722598,
    "pressureSafe": "Safe"
  }
}
`

**Reply by ruspa1978** [reply]
Solved!!! 
The right url is : http://aagsolo/cgi-bin/cgiLastDataJSON
In Lunatico manual is used only http://aagsolo/cgi-bin/cgiLastData without JSON. 
The JSON word was missing.
